tests/controls/gridtest.cpp

```diff
@@ -1635,4 +2224,197 @@ TEST_CASE("GridBlockCoords::SymDifference", "[grid]")
     }
 }
 
+//
+// TestableGrid
+//
+
+void TestableGrid::DoEdit(const EditInfo& edit)
+{
+    m_edit = edit;
+    m_sOriGridAnnotated = ToString();
+
+    if ( edit.isRow )
+    {
+        if ( edit.count < 0 )
+            DeleteRows(edit.pos, -edit.count);
+        else
+            InsertRows(edit.pos, edit.count);
+    }
+    else if ( edit.count < 0 )
+        DeleteCols(edit.pos, -edit.count);
+    else
+        InsertCols(edit.pos, edit.count);
```

Comment:
This is also very trivial, but I'd write it symmetrically to the above, i.e.
```suggestion
    else
    {
        if ( edit.count < 0 )
            DeleteCols(edit.pos, -edit.count);
        else
            InsertCols(edit.pos, edit.count);
    }
```

Of course, if you decide to replace `isRow` with `wxGridDirection`, you'd use a switch on it instead.

Reply:
Thanks. It was like that, will change it back for readability.